packages/odyssey/src/scss/components/_toast.scss

```diff
@@ -136,3 +136,17 @@
     opacity: 0;
   }
 }
+
+@media (prefers-reduced-motion) {
+  .ods-toast {
+    animation-name: ods-toast-in;
+    animation-duration: 300ms;
+    animation-delay: 0s;
+
+    &.is-toast-dismissed {
+      animation-name: ods-toast-out;
+      animation-duration: 300ms;
+      animation-delay: 0s;
+    }
+  }
+}
```

Comment:
This is great, can we think of a high-level solution? Off the top of my head:

```
[class^="ods"] {
  animation: none;
}
```

This would be in some kind of a global style, which may not help. Could be a mixin that we include conditionally in each file?

Just riffin', let me know what you think.

Reply:
For now, I like handling them case-by-case. In some instances, I think gentle animation is less jarring than a sudden appearance. I like the @mixin idea to help with a standard solve, though. Let's keep noodling!